release-notes/index.dita

```diff
@@ -507,6 +507,21 @@
           <xref keyref="pdf-customization"/>, and how to extend the toolkit by
           <xref keyref="adding-plugins"/> and
           <xref keyref="custom-plugins"/>.</p>
+        <p deliveryTarget="html">Information found exclusively on the DITA-OT website or functional website changes
```

Comment:
This section describes docs changes in the initial 3.5 release, so I'll move the content to the 3.5.4 section.